src/gridstack.ts

```diff
@@ -1414,9 +1416,9 @@ export class GridStack {
 
   /** @internal call to write position x,y,w,h attributes back to element */
   protected _writePosAttr(el: HTMLElement, n: GridStackPosition): GridStack {
-    if (n.x !== undefined && n.x !== null) { el.setAttribute('gs-x', String(n.x)); }
+    if (n.x !== undefined && n.x !== null) { el.setAttribute('gs-x', String(n.x)); el.style.setProperty("--gs-x", String(n.x))}

```

Comment:
I don't understand why setting an element style here is used with the rest of the code you have.... something missing.

Reply:
Will analyse it

Reply:
> Will analyse it. I wish you could analyse, test this idea and implement this in the code base. The performance issues will not be noticeable  in most of the cases and we need not to worry about it as far as I know.

Reply:
well I need proof & data, not 'think'